en_US/data-integration/data-bridge-sqlserver.md

```diff
@@ -0,0 +1,337 @@
+# Ingest Data into Microsoft SQL Server
+
+EMQX supports integration with Microsoft SQL Server. You can save client messages and events to Microsoft SQL Server, or record the online status or online/offline of clients by using events to trigger the data update or removal.
+
+{% emqxee %}
+
+::: tip
+
+The data integration with Microsoft SQL Server is currently only supported in EMQX Enterprise 5.0.3 and above.
```

Comment:
The data integration with Microsoft SQL Server is only supported in EMQX Enterprise 5.0.3 and above.